internal/cpp_doc_generator/conf.py

```diff
@@ -0,0 +1,63 @@
+import subprocess, os
+
+project = 'iceoryx2'
+copyright = '2024, ekxide.io'
+author = 'ekxide.io'
```

Comment:
Is this supposed to be the website or company name?

Reply:
It is just the author, in the footer it is noted copyright by ekxide.io. Yeah maybe rewrite it into ekxide IO GmbH